modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase

```diff
@@ -26,6 +26,37 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test DownloadSampleFileOfTheCustomObject {
+		task ("Given a custom object is published") {
+			ObjectDefinitionAPI.createAndPublishObjectDefinition(
+				en_US_label = "student",
+				en_US_plural_label = "students",
+				name = "Student",
+				requiredStringFieldName = "name");
+		}
+
+		task ("And Given object entry is created") {
+			ObjectDefinitionAPI.createObjectEntryWithName(
+				en_US_plural_label = "students",
+				name = "Jessica");
+		}
+
+		task ("And Given I create an import file task with custom object as the entity type") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoImport();
+		}
+
+		task ("When I download a sample file of this entity") {
+			ImportExport.selectEntity(entityType = "C_Student (v1_0 - Liferay Object REST)");
+		}
+
+		task ("Then the sample file download successfully") {
+			Click.clickAt(locator1 = "ImportExport#SAMPLE_FILE_DOWNLOAD_BUTTON");
```

Comment:
Same here, this is "When I download a sample file of this entity", we can try something like WaitForElementPresent(locator1 = "ImportExport#EXECUTION_SUCCESS"); here to make sure it downloads successfully :smile:

Reply:
Thanks for feedback @yanan - About this step, it does not show a success message, when clicking on download, it just downloads to the local machine and does not show anything on the page, for this reason, it would not be possible to confirm if the element is present on the page

Reply:
From my side, I think you can assert the download file have downloaded to your machine(ImportExport.assertFileContent)

Reply:
Hi @pedroamarall , thanks for sharing. Apart from what Kevin shared about the assertFileContent, I think we can also try this

> ImportExport.assertDownloadedFileName()

Reply:
Done, thanks guys